### PR TITLE
Enable https connection on blip server

### PR DESCRIPTION
**Pull request by ragedblp**

### CODE REVIEW

**Comment by ragedblp** [comment]
Waiting for review

**Review by yvon-dblg — APPROVED** [review]
LGTM
But if you want to have a docker image, you need to bump version to an rc.

File: packages/viz/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "d3-shape": "1.2.2",
     "d3-time": "1.0.10",
     "d3-time-format": "2.1.3",
-    "gsap": "3.2.6",
+    "gsap": "3.6.0",
```
Comment (yvon-dblg) [inline packages/viz/package.json:45]:
```suggestion
    "gsap": "^3.6.0",
```

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
package-lock.json need to be fully updated:
- `npm run mrproper`
- `rm package-lock.json`
- `npm install`

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**